app/controllers/report_controller/reports.rb

```diff
@@ -123,7 +123,7 @@ def get_all_reps(nodeid = nil)
         @settings.store_path(:perpage, :list, @items_per_page) # Set the per page setting for this gtl type
       end
 
-      @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? 0 : session["#{x_active_tree}_sortcol".to_sym].to_i
+      @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? ReportController::DEFAULT_SORT_COL : session["#{x_active_tree}_sortcol".to_sym].to_i
```

Comment:
```suggestion
      @sortcol = session["#{x_active_tree}_sortcol".to_sym].nil? ? DEFAULT_SORT_COL : session["#{x_active_tree}_sortcol".to_sym].to_i
```

Since we're within the same controller, you should need to fully-qualify here